packages/core/src/modal/modal.element.ts

```diff
@@ -44,8 +37,9 @@ import { styles } from './modal.element.css.js';
  * @slot cds-modal-header
  * @slot cds-modal-actions
  * @event closeChange - notify when the user has clicked the close button
- * @cssprop --backdrop-opacity
- * @cssprop --backdrop-background
+ * @cssprop --backdrop-opacity - inherited from the internal overlay component
+ * @cssprop --backdrop-background - inherited from the internal overlay component
+ * @cssprop --layered-backdrop-opacity - inherited from the internal overlay component
```

Comment:
May not need these anymore once rebased onto themes with the object tokens http://localhost:6006/?path=/story/foundation-object-styles--page#object-layers